### DISCUSSION TITLE
Digital_Samples.hr2 Greyed Out

### DISCUSSION THREAD

**Discussion by k-srvHs** [publicsamples/Quetzalcoatl#23] [General]
Anyone having this issue where you can't load the Digital_Samples.hr2? I've downloaded it, redownloaded it, but I still can't load it onto the plugin.
<img width="917" alt="digital greyed out" src="https://github.com/user-attachments/assets/518d61ad-308c-46c5-a87d-4a5c854bdbd3">

**Comment by RickTaylor79** [comment]
You can think of the hr files as like multipart zips, you don't actually
need to interact with any file other than hr1 to install the pack.
